umap/settings/base.py

```diff
@@ -39,30 +39,34 @@
     ('ca', 'Catalan'),
     ('cs-cz', 'Czech'),
     ('da', 'Danish'),
-    ('de', 'Deutsch'),
+    ('de', 'German'),
     ('el', 'Greek'),
     ('en', 'English'),
-    ('es', 'Español'),
+    ('es', 'Spanish'),
     ('fi', 'Finnish'),
-    ('fr', 'Francais'),
+    ('fr', 'French'),
     ('gl', 'Galician'),
+    ('he', 'Hebrew'),
```

Comment:
AFAIK, hebrew is a RTL language, and sadly uMap is not ready for that yet :(

Reply:
Ok, I created an issue #752.